doc/openapi-kuadrant-extensions.md

```diff
@@ -1,22 +1,21 @@
 ## OpenAPI 3.0.X Kuadrant Extensions
 
-### Info level kuadrant extension
+### Root level kuadrant extension
```

Comment:
I think this is a very welcome change. I always found it weird, having it under the `info` field.
Well done!